lightbeam/__main__.py

```diff
@@ -117,21 +121,50 @@ def main(argv=None):
         print(f"unknown arguments {unknown_args_str} passed, use -h flag for help")
         exit(1)
     
-    if args.command not in ALLOWED_COMMANDS.values():
-        if args.command is None:
-            logger.error(f"no command provided. Use one of ({command_list}), see -h flag for help")
-        else:
-            logger.error(f"unknown command '{args.command}' passed, use -h flag for help")
-        exit(1)
-    
     if args.version:
         lb_dir = os.path.dirname(os.path.abspath(__file__))
         version_file = os.path.join(lb_dir, 'VERSION.txt')
         with open(version_file, 'r') as f:
             VERSION = f.read().strip()
             print(f"lightbeam, version {VERSION}")
         exit(0)
+    
+    if args.test:
+        tests_dir = os.path.join( os.path.realpath(os.path.dirname(__file__)), "tests" )
+        
+        # Prompt for Ed-Fi API base_url, key, secret (with defaults)!
+        print("Welcome to the lightbeam test suite. This feature is intended for developer use.")
+        print("RUNNING THIS TEST SUITE WILL CAUSE CHANGES TO YOUR ED-FI DATA - USE WITH CAUTION!")
+        base_url = input("Enter your Ed-Fi API's base URL: [\"https://localhost/api\"]:") or "https://localhost/api"
+        client_id = input("Enter your API client ID: [\"populated\"]:") or "populated"
+        client_secret = input("Enter your API client secret: [\"populatedSecret\"]:") or "populatedSecret"
```

Comment:
One of the first things lightbeam does on init is to try to connect to the API... so we have to define that up front. Therefore we begin the test suite by warning the user and then asking them to provide the base URL, client ID, and secret for the API they want to use for the test. The default values work with a typical Docker deployment of the Ed-Fi API.